frontend/src/Metamaps/PasteInput.js

```diff
@@ -29,12 +28,10 @@ const PasteInput = {
       if (e.dataTransfer.files.length > 0) {
         self.handleFile(e.dataTransfer.files[0], coords)
       }
-      // OMG import bookmarks 😍
+      // OMG import bookmarks 😍 (Or just text :P)
```

Comment:
i checked it - you are right, if i drag text e.g. from the input at the top then it imports as an "item"